modules/meeting/app/components/meeting_agenda_items/item_component/show_component.html.erb

```diff
@@ -1,5 +1,5 @@
 <%=
-  grid_layout('op-meeting-agenda-item', tag: :div) do |grid|
+  grid_layout((meeting_open? ? 'op-meeting-agenda-item' : 'closed-meeting op-meeting-agenda-item'), tag: :div) do |grid|
```

Comment:
The first parameter of the `grid-layout` is used for the grid-defintions, so please do not use it for plain classes. Further, `closed-meeting` should be a modifier of the agenda-item, e.g. `op-meeting-agenda-item--closed`